packages/shared_preferences/shared_preferences_windows/pubspec.yaml

```diff
@@ -11,20 +12,21 @@ flutter:
         pluginClass: none
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: '>=2.12.0-259.8.beta <3.0.0'
```

Comment:
Let's stay consistent about using `"`

Why the specific version here, rather than `-0`?

Reply:
I believe pub was not resolving at all, hence changed the version here.

Reply:
I'll try changing and see if ti throws any errors.